.gitignore

```diff
@@ -75,3 +75,5 @@ tags
 /vim-*.patch
 
 /CMakeUserPresets.json
+# ctest artifacts
+Testing
```

Comment:
Still what is this?

Reply:
this will get generated if you mess up and call `ctest` directly without giving it a test-dir (or a preset)

```sh
cd <project-root>
rm -rf Testing
ctest --show-only # there are probably a few different flags that does this as well
test -d Testing && echo "ctest can be stupid"
```

Reply:
Eh, not convincing, if `ctest` generates this it should just be removed instead of being ignored. I'd say skip this.

Reply:
removed by who? it will just be an annoying folder that gets in the way, same as any other cache folder

see: https://github.com/Kitware/CMake/blob/a3252ac53eeb9f0663a9410e3d48340741c348ea/.gitignore#L12

Reply:
also see: https://github.com/neovim/neovim/pull/18674#discussion_r906829680

Reply:
But it's not a cache folder, you said it shows up if you use the wrong command like `ctest --show-only`. If I accidentally write `mkdir random_dir` then the correct solution isn't to add `random_dir` to `.gitignore`, it's to not write wrong commands. I don't see your point here.

Reply:
> But it's not a cache folder, you said it shows up if you use the wrong command like `ctest --show-only`.

it's still a cache folder, just in the wrong directory. when used correctly it shows up in test-dir, in our case it's under `build/`

> If I accidentally write `mkdir random_dir` then the correct solution isn't to add `random_dir` to `.gitignore`, it's to not write wrong commands.

Your example doesn't apply here.

- we're adding a new tool that is known to generate a folder that should not be tracked
- this folder has a very _specific_ name, it's always _Testing_
- this makes it a perfect candidate to be added to `.gitignore`
- it will help users realize that this folder is safe to ignore/delete as well
- It's not worth it to try to figure out why and how `ctest` has this bug/feature/behavior
- no file/folder called _Testing_ is currently tracked, and I doubt there will ever be one

so all in all this ranks an easy 9/10 on my scale for QoL/effort